Tasks/DotnetCLIRestore/task.json

```diff
@@ -0,0 +1,159 @@
+{
+    "id": "9fac244b-8d7c-4d8e-a003-2097daa3270f",
+    "name": "DownloadGitHubNugetPackage",
+    "friendlyName": "Download Nuget GitHub Packages",
+    "description": "Restore your nuget packages using dotnet CLI",
+    "author": "Microsoft Corporation",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/build/dotnet-core-cli",
```

Comment:
I hope this will be changed

Reply:
Since I don't have a ready doc yet, I will change this to the generic doc page for tasks.